### PR TITLE
fixes #4 (Use of bind polyfill is required for testing of anything …

### LINKED ISSUE #4: Use of bind polyfill is required for testing of anything that uses angular-translate-once
Looks like unit tests of any code that uses angular-translate-once fail with

```
TypeError: 'undefined' is not a function (evaluating 'TranslateOnceAttributeDirective.bind(undefined, attribute)')
```

at line https://github.com/ajwhite/angular-translate-once/blob/master/src/translate-once.js#L16 unless your Function.prototype.bind polyfill is added, which is not ideal for a third party library.

Do you have another strategy of avoiding above error while testing? why use a function that requires a polyfill?

### PR DESCRIPTION
**Pull request by runderworld**
…that uses angular-translate-once)

Let's use angular's built-in bind() polyfill so as not to require any added polyfills.

### CODE REVIEW

File: src/translate-once.js [hunk 1]
```diff
@@ -13,7 +13,7 @@
   createDirective = function (attribute) {
     var namedDirective = getNamedDirectiveFromAttribute(attribute);
     angular.module(MODULE_NAME).directive(namedDirective, ['$parse', '$translate',
-    TranslateOnceAttributeDirective.bind(undefined, attribute)]);
+    angular.bind(undefined, TranslateOnceAttributeDirective, attribute)]);
```
Comment (atticoos) [inline src/translate-once.js:16]: I feel so silly for not realizing [this was supported by the angular library](https://docs.angularjs.org/api/ng/function/angular.bind). Thanks for this!!

**Comment by atticoos** [comment]
:+1: :100:

File: src/translate-once.js (hunk 1 above)
Comment (runderworld) [inline src/translate-once.js:16]: No problem, thanks for sharing this library!

And yeah, my teammates and I have always gone the route of including a file of essential polyfills for testing with PhantomJS—`angular.bind()` seems so verbose and less readable when sitting next to a usage of `Function.prototype.bind()`. But in this case, why not!

Comment (atticoos) [inline src/translate-once.js:16]: Company products, sure! 

Open source projects... :no_good:

**Pull request merged**